src/main/java/servlets/UploadReceiptServlet.java

```diff
@@ -244,8 +244,9 @@ private Entity analyzeReceiptImage(BlobKey blobKey, HttpServletRequest request)
       receipt.setProperty("timestamp", timestamp);
     });
 
-    // TODO: Replace with parsed price.
-    receipt.setProperty("price", FormatUtils.roundPrice(request.getParameter("price")));
+    // Set the price if it was parsed.
+    results.getPrice().ifPresent(
+        price -> { receipt.setProperty("price", FormatUtils.roundPrice(price)); });
```

Comment:
nitnit: Since lambda function is only one line, I believe you can do
`price -> receipt.setProperty("price", FormatUtils.roundPrice(price))`
without the brackets.